hc/front/tests/test_add_pushbover.py

```diff
@@ -1,40 +1,79 @@
+        ascertaining a new session. Priority of normal is passed. User is
+        redirected. Channel should contain one object.
+        """
         self.client.login(username="alice@example.org", password="password")
 
         session = self.client.session
         session["po_nonce"] = "n"
         session.save()
 
         params = "pushover_user_key=a&nonce=n&prio=0"
-        r = self.client.get("/integrations/add_pushover/?%s" % params)
-        assert r.status_code == 302
+        get_request = self.client.get("/integrations/add_pushover/?%s" % params)
+        assert get_request.status_code == 302
 
         channels = list(Channel.objects.all())
         assert len(channels) == 1
         assert channels[0].value == "a|0"
 
     @override_settings(PUSHOVER_API_TOKEN=None)
     def test_it_requires_api_token(self):
+        """Test that pushover_api_token is checked if it exists.
+        User logs in with no pushover token.
+        """
         self.client.login(username="alice@example.org", password="password")
-        r = self.client.get("/integrations/add_pushover/")
-        self.assertEqual(r.status_code, 404)
+        r_with_no_token = self.client.get("/integrations/add_pushover/")
+        self.assertEqual(r_with_no_token.status_code, 404)
 
     def test_it_validates_nonce(self):
+        """Test that nonce is validated before a pushover channel is added.
+        Lack of nonce forbids the user from access.
+        """
         self.client.login(username="alice@example.org", password="password")
 
         session = self.client.session
         session["po_nonce"] = "n"
         session.save()
 
         params = "pushover_user_key=a&nonce=INVALID&prio=0"
-        r = self.client.get("/integrations/add_pushover/?%s" % params)
-        assert r.status_code == 403
+        r_with_invalid_nonce = self.client.get("/integrations/add_pushover/?%s" % params)
+        assert r_with_invalid_nonce.status_code == 403
 
     ### Test that pushover validates priority
+    def test_it_validates_priority(self):
+        """Test parameter priority is checked before adding a pushover.
+        Pushover priority parameter should range as:
+        -2 -- Lowest priority
+        -1 -- Low priority
+        0 -- Normal
+        1 -- High priority
+        2 -- Emergency priority
+        Parameter outside above range should return bad request status code.
+        """
+
+        self.client.login(username="alice@example.org", password="password")
+        session = self.client.session
+        session["po_nonce"] = "n"
+        session.save()
+        invalid_prio_param_1 = "pushover_user_key=a&nonce=n&prio=-3"
+        invalid_prio_param_2 = "pushover_user_key=a&nonce=n&prio=3"
+        r_with_invalid_prio = self.client.get("/integrations/add_pushover/?%s"
+                                              % invalid_prio_param_1)
+        assert r_with_invalid_prio.status_code == 400
```

Comment:
other asserts use `self.assert`. When you decide on a particular convention, its good to stick with it. Please use `self.assert` for this.